app/views/seller/orders/show.html.erb

```diff
@@ -1,4 +1,92 @@
+        </tr>
+        <tr>
+          <td><strong>Display Name:</strong><br /><%= @order.user.display_name %></td>
+        </tr>
+        <tr>
+          <td><strong>Ship To Address:</strong>
+            <br /><%= @order.user.street_1 %>
+            <% if @order.user.street_2 %>
+            <br /><%= @order.user.street_2 %>
+            <% end %>
+            <br /><%= @order.user.city %> <%= @order.user.state %>, <%= @order.user.zipcode %>
+          </td>
+        </tr>
+        <tr>
+          <td><strong>Ordered at:</strong><br /><%= @order.created_at.strftime("%e %b %Y %H:%M:%S%p") %></td>
+        </tr>
+      </table>
+    </div>
+  </div>
+
+  <div class="col-md-9 event-page-ticket-listing user-dash-sidebar">
+    <div class="event-page-inner">
+      <h3>Item Details</h3>
+      <table class="table table-hover event-tickets">
+        <thead>
+          <tr>
+            <th></th>
+            <th></th>
+            <th>Title</th>
+            <th>Description</th>
+            <th>Location</th>
+            <th>Section</th>
+            <th>Row</th>
+            <th>Seat</th>
+            <th>Delivery</th>
+            <th>Seller</th>
+            <th colspan=2>Price</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @order.items.each do |item| %>
+          <tr class="event-row">
+            <td></td>
+            <td>
+              <div class="event-date-group">
+                <p class="event-month"><%= item.event.month %></p>
+                <p class="event-date"><%= item.event.day_of_month %></p>
+                <p class="event-day"><%= item.event.day_of_week %></p>
+              </div>
+            </td>
+            <td class="vert-align"><%= link_to(item.event.title, event_path(item.event)) %></td>
+            <td class="vert-align"><%= item.event.description %></td>
+            <td class="vert-align"><%= link_to(item.event.venue.name, venue_path(item.event.venue)) %></td>
+            <td class="vert-align"><%= item.section %></td>
+            <td class="vert-align"><%= item.row %></td>
+            <td class="vert-align"><%= item.seat %></td>
+            <td class="vert-align delivery-meth">
+              <p><%= item.delivery_method %></p>
+              <% if item.delivery_method == "electronic" %>
+              <p><%= link_to "download", item.ticket.url(:original, false) %></p>
+              <% end %>
+            </td>
+            <td class="vert-align"><%= link_to(item.seller, seller_store_path(item.seller)) %></td>
+            <td class="vert-align"><%= item.dollar_amount %></td>
+          </tr>
+          <% end %>
+        </tbody>
+        <tfoot>
+          <tr>
+            <td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td>
+            <td> <h3>Total:</h3> </td>
+            <td> <h3 class="total_price"> <%="#{@order.total_price}" %></h3></td>
+          </tr>
+        </tfoot>
+      </table>
+    </div>
+  </div>
 </div>
```

Comment:
I dont' think we even use this view anymore?? This is the same as the seller_order_path. We can delete this tomorrow is we are all sure it's not needed.